Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -2193,6 +2199,26 @@ extension ALKConversationViewController: ALKConversationViewModelDelegate {
         }
     }
     
+    private func isZendeskConversation(channel : ALChannel?) -> Bool{
```

Comment:
this restriction is for agent app. Will it affect anything on SDK ? I don't see agent app check here

Reply:
isZendeskConversation() is only called inside checkRestricted() has a agent app check. So it will only be called inside agent app